_posts/python/basic/2017-02-24-python-basic-index.html

```diff
@@ -10,6 +10,8 @@
 thumbnail: thumbnail/mixed.jpg
 page_type: example_index
 dash_url: https://dash-simple-apps.plotly.host/dash-basicplot/frameBorder=0
```

Comment:
We're going to need to fix this in order to avoid showing the code page of the app twice. It looks like Divya made some changes in the app source and now the code page is returned if `pathname` isn't null.

![code-example-dash-app](https://user-images.githubusercontent.com/41019918/57792194-6dfb2800-770c-11e9-94d5-23b698a6ec66.png)

```suggestion
dash_url: https://dash-simple-apps.plotly.host/dash-basicplot/?frameBorder=0
```

Reply:
what is this `frameBorder=0` thing intended to do? it's not parsed or accepted by the dash apps themselves AFAICT

Reply:
it looks like an `<iframe>` HTML attribute to me